projects/demo-maps/src/app/route-components/route-example-olperformance/fetchbutton/fetchbutton.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { LayersService, VectorLayer } from '@dlr-eoc/services-layers';
+import { Fill as olFill, Stroke as olStroke, Style as olStyle } from 'ol/style';
+import simplify from '@turf/simplify';
```

Comment:
can you try to use openLayers for this as well [ol/geom/SimpleGeometry](https://openlayers.org/en/latest/apidoc/module-ol_geom_SimpleGeometry-SimpleGeometry.html)

Reply:
Thank you! Will do - update should be here by Monday.